apps/core-app/src/main/modules/box-tool/addon/preview/preview-provider.test.ts

```
@@ -0,0 +1,94 @@
+import { beforeEach, describe, expect, it, vi } from 'vitest'
+import type { IExecuteArgs } from '@talex-touch/utils'
```

ESLint

[error] Expected "`@talex-touch/utils`" (type) to come before "vitest" (external).

(perfectionist/sort-imports)